mlir/lib/Analysis/PresburgerSet.cpp

```diff
@@ -318,6 +318,52 @@ bool PresburgerSet::findIntegerSample(SmallVectorImpl<int64_t> &sample) {
   return false;
 }
 
+PresburgerSet PresburgerSet::coalesce() {
+  assert(nSym == 0 && "symbolic sets are not yet supported by coalesce");
```

Comment:
I think the same code should work with sets with symbols, so this shouldn't be necessary.